Pipfile.lock

```diff
@@ -405,27 +397,13 @@
             "markers": "python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'",
             "version": "==2.10"
         },
-        "importlib-metadata": {
```

Comment:
@robeden It seems dropping this caused our UT to fail, but actual scanning (`integration-tests` job) completed successfully. Can you add it to the dev-packages in our Pipfile?

Reply:
Thanks, I reverted. The lock didn't need to be updated.